k.makoyid/oop_code/Visualizer.py

```diff
@@ -0,0 +1,26 @@
+import matplotlib.pyplot as plt
+
+
+class Visualizer:
+    @staticmethod
+    def plot_results(train_losses, train_accuracies, test_losses, test_accuracies):
+        plt.figure(figsize=(12, 8))

```

Comment:
There is no need in class if it contains only static methods, especially this one